frontend/src/pages/frontPage/Panel.jsx

```diff
@@ -1,13 +1,14 @@
 import Nav from "../../components/layout/Nav";
 import { Outlet } from "react-router-dom";
-
+import SignIn from "../../components/layout/signIn";
 import SignUp from "../../components/layout/SignUp";
 const Panel = (props) => {
   const { isLoggedIn } = props;
   return (
     <>
       <Nav isLoggedIn={isLoggedIn} />
-      <SignUp isLoggedIn={isLoggedIn} />
```

Comment:
The other PR I made a request to only show the popup when I click "register", which we should also do here when I click "login". Let's fix it in that branch, merge those changes, then rebase this branch and do the same thing.